profiles/common/modules/custom/media_avportal/includes/MediaAvportalStreamWrapper.inc

```diff
@@ -63,15 +62,13 @@ class MediaAvportalStreamWrapper extends MediaReadOnlyStreamWrapper {
       else {
         $url = 'http://ec.europa.eu/avservices/cfc/video.cfc?method=getVideoDetails&ref=' . $video_id . '&format=json';
         $response = drupal_http_request($url);
-        if ($response->data == 200) {
+
+        if (!isset($response->error)) {
           $video_info = drupal_json_decode($response->data);
```

Comment:
Hello @verbruggenalex,

The title and thumb are not actually required and might be empty in a lot of cases.
See e.g.: http://ec.europa.eu/avservices/cfc/video.cfc?method=getVideoDetails&ref=I053681&format=json
So I would keep it like that. This is the easiest (most simple) fix IMO.

But I did my own review of media_avportal here in case you want to go deeper: https://github.com/vever001/platform-dev/pull/1
That might be out of scope.

Regards,
Hervé